src/js/helpers/xmppEventHandlers.js

```diff
@@ -236,20 +243,17 @@ module.exports = function (client, app) {
     });
 
     client.on('avatar', function (info) {
-        var contact = me.getContact(info.jid);
+        var contact = me.getContact(info.jid) || me.isMe(info.jid);
         if (!contact) {
-            if (me.isMe(info.jid)) {
-                contact = me;
-            } else {
-                return;
-            }
```

Comment:
Nope, it's wrong again.

You've written the following:
```
var contact = (me.getContact(info.jid) || me.isMe(info.jid)) ? me : false;
```

But the original was:
```
var contact = me.getContact(info.jid) || (me.isMe(info.jid) ? me : undefined);
```

Remember the operator priority.